charts/dealer/templates/graphql-service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if (index .Values "fake-galoyapi").enabled }}
```

Comment:
I think the file names should also have `fake` in them.

Reply:
Actually they should be called the same as the setting in values.yml. So this should be `fake-galoyapi-service.yml` etc.